### ISSUE TITLE
Grams: Gradient Descent with Adaptive Momentum Scaling

### ISSUE THREAD

**Issue by 67372a** [kozistr/pytorch_optimizer#317]
Another momentum update modifier like cautious, seems promising.
Paper: https://arxiv.org/abs/2412.17107
Code: No official repo as far as I know, but the code is simple:
```
update.copy_(torch.sign(grad) * update.abs())
```

**Comment by kozistr** [comment]
thanks!
